roles/sap_hana_install/defaults/main.yml

```diff
@@ -90,7 +90,7 @@ sap_hana_install_components: 'all'
 #sap_hana_install_hdblcm_extraargs: '--ignore=check_diskspace,check_min_mem'
 
 # Instance details
-sap_hana_install_sid:
+sap_hana_install_sid: "{{ sap_hana_sid }}"
 sap_hana_install_number:
```

Comment:
Please also add "{{ sap_hana_instance_number }}" here, as in the other three roles.